webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -126,26 +125,25 @@ class Test_generate_create_bug_url:
         + "comment=%(description)s&"
         + "format=__default__"
     )
+    CRASH_ID = "70dda764-a402-4ca3-b806-c38dd0240328"
 
-    @staticmethod
-    def _create_report(**overrides):
+    def _create_report(self, **overrides):
         default = {
             "signature": "$&#;deadbeef",
-            "uuid": "00000000-0000-0000-0000-000000000000",
```

Comment:
This is not a valid crash id. I generated a valid crash id and switched everything to use that. This will save future us from having to deal with consequences of invalid test data.